argus/action_manager/windows.py

```diff
@@ -529,12 +537,154 @@ def prepare_config(self, cbinit_conf, cbinit_unattend_conf):
         """
         super(WindowsNanoActionManager, self).prepare_config(
             cbinit_conf, cbinit_unattend_conf)
-        cbinit_conf.set_conf_value("stop_service_on_exit", False)
+        for conf in (cbinit_conf, cbinit_unattend_conf):
+            conf.conf.remove_option("DEFAULT", "logging_serial_port_settings")
+            conf.set_conf_value("stop_service_on_exit", False)
+
+    def get_installation_script(self):
+        """Get instalation script for CloudbaseInit."""
+        self.download_resource("windows/nano_server/installCBinit.ps1",
+                               self._INSTALL_SCRIPT)
+
+    def _unzip(self, zip_file, out_file, force=False):
+        """Unzip the file.
+
+        :param zip_file:
+            Path to zip file.
+        :param out_file:
+            Path to output file from unzipping process.
+        :param force:
+            If we should override the output file or not.
+        """
+        exists = self.exists(out_file)
+        if exists and not force:
+            raise exceptions.ArgusCLIError("Output file path already exists.")
+        elif exists:
+            (self.rmdir if self.is_dir(out_file) else self.remove)(out_file)
+
+        cmd = ("[System.IO.Compression.ZipFile]"
+               "::ExtractToDirectory('{}', '{}')").format(zip_file, out_file)
+
+        self._client.run_command_with_retry(
+            cmd, command_type=util.POWERSHELL)
+
+    @staticmethod
+    def _get_installer_name():
+        """Get Cloudbase-Init installer name."""
+        if CONFIG.argus.build != "Beta":
```

Comment:
Shouldn't this be `CONFIG.argus.build == "Beta"` ? From the note you left you seem to be taking this aproach for the URL formatting when the build is `Beta`